### PR TITLE
Admin rejects a project

### PR DESCRIPTION
**Pull request by zspencer**
Resolves https://github.com/wecohere/nourish.party/issues/58

### CODE REVIEW

**Comment by zspencer** [comment]
@bhaibel - dry it up or leave it be?

File: app/models/project.rb [hunk 1]
```diff
@@ -12,10 +12,19 @@ def approve
...
+    update(status: "rejected")
+  end
+
   def approved?
     status == "approved"
   end
 
+  def rejected?
+    status == "rejected"
+  end
```
Comment (zspencer) [inline app/models/project.rb]: I feel like I should be using acts_as_state_machine or something similar for this?

File: app/views/project_rejections/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h1>Approve <%= pending_project.title %>?</h1>
+<p><%= pending_project.summary %></p>
+<%= form_for(project_rejection,
+             url: pending_project_project_rejections_url(pending_project),
+             html: { data: { type: "project-status-change-form" } }) do |form| %>
+  <div class="fields">
+    <%= form.label :reason, t('rejecting_project.reason_field') %>
```
Comment (zspencer) [inline app/views/project_rejections/new.html.erb]: BLOCKIGN - Fill these into the en.yml before merge

File: app/models/project.rb (hunk 1 above)
Comment (bhaibel) [reply app/models/project.rb]: Let's not use acts_as_state_machine, it's a one-way door to callback hell. This is a productive ugly and the correct pattern will emerge if we leave it be.

**Comment by bhaibel** [comment]
Leave it be. This all looks perfectly reasonable for "we're doing halfway event sourcing and we don't know what the patterns are yet."

**Pull request merged**